test/unit_Polytope.jl

```diff
@@ -246,6 +246,10 @@ for N in [Float64]
     q = VPolytope([N[0, 1], N[0, 2]])
     @test N[0, 1//2] ∉ q
 
+    # creation from a matrix (each column is a vertex)
+    pmat = VPolytope(copy(N[0 0; 1 0; 0 1]'))
```

Comment:
Why the `copy` and the transpose?

Reply:
Each vertex is a column (hence the transpose); the `copy` is to materialize the lazy transpose, this is standard.

Reply:
Okay, I thought that you could as well write the final matrix directly, but if it helps in reading... (for me it does not :wink:)

Reply:
This is what comes natural to me, i don't know .. :) I have to think about where to put the commas, if i have to write the final matrix directly.

Reply:
You can add a new example in the docstrings if you'd prefer, examples never hurt..